$NetBSD: patch-ad,v 1.2 2006/06/13 13:24:36 dsainty Exp $

The R65C02 is a generally more capable processor than the M6502, and a BBC B
can run almost all software with a R65C02 replacing its standard CPU.  In
addition, the R65C02 allows running some software intended for the Master
series of microcomputer, making the R65C02 the more interesting CPU to
emulate.

Since patch-ac modifies the OS1.2.rom on the fly, it is no longer necessary to
patch the ROM externally for EMUL_FS, so keep OS_ROM the same.

--- src/Config.h.orig	2002-01-16 04:46:43.000000000 +1300
+++ src/Config.h	2006-06-04 12:44:23.000000000 +1200
@@ -290,8 +290,8 @@
  * Processor type
  */
 
-#define		M6502
-#undef		R65C02
+#undef		M6502
+#define		R65C02
 #undef		R65C12
 
 /*
@@ -344,11 +344,7 @@
  * Names of ROMs to load by default
  */
   
-#ifdef	EMUL_FS
-#define		OS_ROM			"OS1.2p1.rom"
-#else
 #define		OS_ROM			"OS1.2.rom"
-#endif
 
 #ifdef	M6502
 #define		LANG_ROM		"BASIC2.rom"
